docs/xcm/building-with-xcm/xc-remote-transact.md

```diff
@@ -2,57 +2,129 @@
 sidebar_position: 6
 ---
 
-# Remote Transact via Smart Contracts
+# Remote Transact via XCM
 
-XCM's `Transact` instructions allows sender to execute arbitrary call in the destination chain. This allows us to execute remote transactions
-in other chains, in case they support or allow such functionality.
+## Feature Overview
 
-E.g. a user could send `Transact` instruction from **Astar** to **Polkadot** that will transfer some `DOT` from user's derived account on **Polkadot** to an arbitrary receiver account on **Polkadot**. The end user does't directly interact with **Polkadot** chain, but can still change its state. It's important to note that this is just an example - any call that can be interpreted by the remote (destination) chain can potentially be sent and executed.
+XCM's `Transact` instructions allows the sender to execute arbitrary calls in the destination chain. This feature is extremely useful since it allows us to control our account on a remote chain.
```

Comment:
```suggestion
XCM's `Transact` instructions allow the sender to execute arbitrary calls on the destination chain. This feature is extremely useful since it allows us to control our account on a remote chain.
```